package proto_ff;

import "yd_fieldoptions.proto";


message rechargerechargepresentDesc
{
	optional int32 Num = 1[(yd_fieldoptions.field_cname) = "数量"];
	optional int64 Id = 2[(yd_fieldoptions.field_cname) = "id"];
}

message rechargerechargechannelDesc
{
	optional int32 id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional string point = 2[(yd_fieldoptions.field_cname) = "计费点", (yd_fieldoptions.field_bufsize) = 32];
}

message rechargerecharge
{
	optional int32 id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional string name = 2[(yd_fieldoptions.field_cname) = "名称", (yd_fieldoptions.field_bufsize) = 42];
	optional string defaultPoint = 3[(yd_fieldoptions.field_cname) = "默认计费点", (yd_fieldoptions.field_bufsize) = 32];
	optional int32 type = 4[(yd_fieldoptions.field_cname) = "充值类型"];
	optional string unit = 5[(yd_fieldoptions.field_cname) = "单位", (yd_fieldoptions.field_bufsize) = 32];
	optional string beizhu = 6[(yd_fieldoptions.field_cname) = "备注", (yd_fieldoptions.field_bufsize) = 103];
	optional int64 quantity = 7[(yd_fieldoptions.field_cname) = "获取档位"];
	optional int32 itemNum = 8[(yd_fieldoptions.field_cname) = "获得物品的数量"];
	optional int64 multipleTimes = 9[(yd_fieldoptions.field_cname) = "重复次数"];
	optional int64 multipleReward = 10[(yd_fieldoptions.field_cname) = "重复奖励"];
	optional int32 zuanshitype = 11[(yd_fieldoptions.field_cname) = "获取类型"];
	optional int32 vipExp = 12[(yd_fieldoptions.field_cname) = "vip经验"];
	optional int32 vipExpLimit = 13[(yd_fieldoptions.field_cname) = "vip经验限制"];
	optional int32 activityCount = 14[(yd_fieldoptions.field_cname) = "运营活动计数"];
	optional int32 HactivityCount = 15[(yd_fieldoptions.field_cname) = "高级运营活动计数"];
	optional int32 price = 16[(yd_fieldoptions.field_cname) = "售价"];
	optional int32 fristpresent = 17[(yd_fieldoptions.field_cname) = "首充赠送数量"];
	optional int32 presenttype = 18[(yd_fieldoptions.field_cname) = "赠送类型"];
	optional string icon = 19[(yd_fieldoptions.field_cname) = "图标", (yd_fieldoptions.field_bufsize) = 32];
	optional int32 preRecharge = 20[(yd_fieldoptions.field_cname) = "是否允许金券充值"];
	repeated rechargerechargepresentDesc present = 21[(yd_fieldoptions.field_cname) = "充值赠送物品", (yd_fieldoptions.field_arysize) = 2];
	repeated rechargerechargechannelDesc channel = 22[(yd_fieldoptions.field_cname) = "渠道", (yd_fieldoptions.field_arysize) = 4];
}


message Sheet_rechargerecharge
{
	repeated rechargerecharge rechargerecharge_List = 1[(yd_fieldoptions.field_arysize)=1338];
}
